src/nvim/drawline.c

```diff
@@ -616,16 +616,14 @@ int win_line(win_T *wp, linenr_T lnum, int startrow, int endrow, bool nochange,
   int c_extra = NUL;                  // extra chars, all the same
   int c_final = NUL;                  // final char, mandatory if set
   int extra_attr = 0;                 // attributes when n_extra != 0
-  static char_u *at_end_str = (char_u *)"";  // used for p_extra when displaying
-                                             // curwin->w_p_lcs_chars.eol at
-                                             // end-of-line
+  static char *at_end_str = "";  // used for p_extra when displaying curwin->w_p_lcs_chars.eol at end-of-line
```

Comment:
Hmm, this line is 109 character long. Why isn't there a lint failure?

Reply:
I've removed it. I thought uncrustify took care of that. Apparently there's a separate `cmt_width` option that we're not using.